hikey.mk

```diff
@@ -134,6 +134,7 @@ endef
 
 .PHONY: edk2
 edk2:
+	sed -i 's/\(^DEFINE GCC_ALL_CC_FLAGS.*-Wno-array-bounds\) -include/\1 -Wno-stringop-overflow -include/' $(EDK2_PATH)/BaseTools/Conf/tools_def.template
```

Comment:
Does this work even when compiling multiple times?

Reply:
Yes. The regex won't match the second time so `-Wno-stringop-overflow` is added only once.